classes/stopper/stopper.js

```diff
@@ -0,0 +1,48 @@
+const Time = require('../time/time');
+const {
+    MAX_STOPPER_SECONDS,
+    MIN_STOPPER_SECONDS,
+} = require('../stopper/utils/consts');
+
+// check what's the time limits of stopper?
+
+class Stopper extends Time {
```

Comment:
in the stopper class, should have also stop method , 
todo: implement stop method